docs/aerodromes/Alice.md

```diff
@@ -29,16 +29,16 @@ For simulation purposes, visual separation is assumed to exist below the cloud b
 Departures from YBAS in to ASP Class C will be coordinated when ready for departure.
 
 !!! example
-    **AS ADC** -> **ASP**: "Next, QFA797"  
-    **ASP** -> **AS ADC**: "QFA797"
+    <span class="hotline">**AS ADC** -> **ASP**</span>: "Next, QFA797"  
```

Comment:
did we decide we were going to go with coldlines for initial calls? some others below may need to be fixed

Reply:
It's a bit messy trying to build a standard, but I went with hot for all next calls given that they're a quick pop up sort of thing (and the FDR activation replaces the taxi call)